spec/rubocop/result_cache_spec.rb

```diff
@@ -200,6 +200,14 @@ def abs(path)
         cache.save(offenses)
       end
     end
+
+    context 'when the @path is not writable' do
+      let(:cache_root) { '/permission_denied_dir' }
+
+      it 'doesn\'t raise an exception' do
+        cache.save([])
```

Comment:
Perhaps even:

```ruby
expect { cache.save([]) }.not_to raise_error
```

🙂

Reply:
Thank you! It's my first time with RSpec. 👍